env.example

```diff
@@ -66,3 +66,9 @@ export DEPLOY_VERSION=v3.11
 
 # Path to the request to send to the OSA RP.
 #export MANIFEST=test/manifests/normal/create.yaml
+
+# Path to the internal config blob written by the fake rp
+export CONFIG_BLOB_PATH=${PWD}/_data/containerservice.yaml
+
+# Path to a directory where openshift development files should be written
+export HELPERS_PATH=${PWD}/_data/_out/
```

Comment:
It's unclear these envs add any value or are going to be tweaked ever.

Reply:
+1 - suggest to remove

Reply:
they probably need to be consts in the code somewhere